docs/user/security/audit-logging.asciidoc

```diff
@@ -89,9 +88,12 @@ Refer to the corresponding {es} logs for potential write errors.
 | `failure` | User is not authorized to create a connector.
 
 .2+| `alert_create`
-| `unknown` | User is creating an alert rule.
```

Comment:
Not really part of this PR but decided to fix the docs for these since we went with "alert_*" rather than "alert_rule_*" in the end